microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -264,7 +274,7 @@ To get more comfortable with the **Performance** tool, practice profiling your p
 
 If you have any questions about your results, use the **Send Feedback** icon, press **Alt+Shift+I** (Windows, Linux) or **Option+Shift+I** (macOS), or [file an issue on the MicrosoftEdge / DevTools repo](https://github.com/MicrosoftEdge/DevTools/issues):
 
-![The **Feedback** icon in the Microsoft Edge DevTools](./index-images/evaluate-performance-feedback-icon.png)
+![The **Feedback** icon in the Microsoft Edge DevTools](./index-images/feedback-icon.png)
```

Comment:
in this rendered section
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/?branch=pr-en-us-2814#next-steps
the png was confusing.  Give a lead-in matching the red box that's on the Feedback icon/button.